### DISCUSSION TITLE
能否添加下载指定日期历史弹幕的功能

### DISCUSSION THREAD

**Discussion by anatoliux2233** [the1812/Bilibili-Evolved#1553] [功能建议 / Ideas]
首先给大佬拜年！
不是biliplus全弹幕装填的意思，是在网页播放器中选择加载某一天的历史弹幕后下载该指定日期的弹幕，目前的状况是在播放器中加载历史弹幕后下载弹幕，内容还是最近一天的最新弹幕。
另外我在使用历史弹幕接口时无法加载，显示“本视频弹幕功能已被关闭”，实际上网页端是可以正常加载的，不知https://api.bilibili.com/x/v2/dm/history?type=1&date=xxxx-xx-xx&oid=xxxxxxxxx 这个端口是否还能正常使用？（以《宝石幻想：光芒重现》第十二集2020年10月7日的弹幕为例）
![screencapture-api-bilibili-x-v2-dm-history-2021-02-13-14_11_23](https://user-images.githubusercontent.com/38915367/107843660-18cfd700-6e08-11eb-8562-a32d8080a215.png)

**Comment by the1812** [comment]
按理来说那个 API 是可以这么用的, 我可以试试让下载弹幕跟随当前历史弹幕的日期

**Reply by anatoliux2233** [reply]
非常感谢
